network/src/outbound/connection_writer.rs

```diff
@@ -73,12 +73,6 @@ impl ConnWriter {
             self.writer.write_all(&self.buffer[..encrypted_len]).await?;
         }
 
-        // If message is a `SyncBlock` message, log it as a trace.
-        match payload {
-            Payload::SyncBlock(_) => trace!("Sent a '{}' message to {}", payload, self.addr),
-            _ => debug!("Sent a '{}' message to {}", payload, self.addr),
```

Comment:
I'm assuming we're avoiding the log here due to this call being "hot"? However, we should still log this someplace, possibly when the message is sent to the connection's `Sender`.

Reply:
no this was a rebase failure, will readd